src/Sylius/Behat/Context/Setup/ShopSecurityContext.php

```diff
@@ -80,5 +83,13 @@ public function iAmLoggedInCustomerByUsingRememberMeOption(): void
         $this->userRepository->add($user);
 
         $this->securityService->logInWithRememberMe($user);
+
+        $this->storeUserToken($user);
+    }
+
+    /** Set the token in the shared storage to have access both in UI and API to resources created in the setup context */
+    private function storeUserToken(UserInterface $user): void
```

Comment:
```suggestion
    private function saveUserToken(UserInterface $user): void
```
🤷 
Although, could we add the `user` saving in here as well? Would be a bit cleaner